### PR TITLE
MSFT-22

### PR DESCRIPTION
**Pull request by r0hit-gupta**
Changes for https://github.com/topcoder-platform/community-app/issues/2745

Based on https://marvelapp.com/c7h2a6h/screen/59648056/handoff

### CODE REVIEW

File: src/styles/_mixins/variables.scss [hunk 1]
```diff
@@ -45,14 +45,14 @@ $tc-dark-blue-30: #cee6ff;
 $tc-dark-blue-10: #f4f9ff;
 
 /* Light Blue - design color code. */
-$tc-light-blue-110: #169ed9;
+$tc-light-blue-110: #0076a5;
```
Comment (veshu) [inline src/styles/_mixins/variables.scss]: @r0hit-gupta I can't find this color on palette

Comment (veshu) [reply src/styles/_mixins/variables.scss]: This color should be used 0B71E6

**Comment by codeMinter** [comment]
@r0hit-gupta use the color from the palette https://marvelapp.com/c7h2a6h/screen/59648056/handoff

In case you can't find one in palette tag `Darak` on the ticket and she will help with one as replacement to use.

File: src/styles/_mixins/variables.scss (hunk 1 above)
Comment (r0hit-gupta) [reply src/styles/_mixins/variables.scss]: @veshu color #0B71E6 is not on the palette either.

Comment (veshu) [reply src/styles/_mixins/variables.scss]: @r0hit-gupta I think we can close this PR, please use variable `$tc-light-blue-110` color in the app. We shouldn't be needing this PR, right?

Comment (r0hit-gupta) [reply src/styles/_mixins/variables.scss]: @veshu I am already using the variable `$tc-light-blue-110` in the app. This PR won't be needed anymore after #19 is merged.

Comment (veshu) [reply src/styles/_mixins/variables.scss]: Ok I am closing this PR, please make sure your issue reference this PR, so reviewer would accept the issue. Thanks

**Pull request closed without merging**